core/templates/dev/head/components/AnswerGroupEditorDirective.js

```diff
@@ -28,20 +28,25 @@ oppia.directive('answerGroupEditor', [
         getOnSaveAnswerGroupRulesFn: '&onSaveAnswerGroupRules',
         getOnSaveAnswerGroupCorrectnessLabelFn: (
           '&onSaveAnswerGroupCorrectnessLabel'),
+        onSaveContentIdsToAudioTranslations: '=',
         outcome: '=',
+        correctnessFeedbackEnabled: '=',
         suppressWarnings: '&',
-        rules: '='
+        rules: '=',
+        isQuestion: '&',
+        addState: '=',
+        getStateNames: '='
```

Comment:
Could also be &: 

getStateNames: '&stateNames'

Reply:
Done, moved this to EditorStateService.